sunpy/tests/test_models.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import time
+import pathlib
+from astropy.table import QTable
+import sunpy.sun.models  as sun_models  
+
+_MODEL_DATA_DIR = pathlib.Path(sun_models.__file__).parent / "data"
+
+
+_MODELS = {
+    "chromosphere_avrett_loeser_2008": _MODEL_DATA_DIR / "chromosphere_avrett_Loeser_2008_model.ecsv",
+  
+}
```

Comment:
What is the reason for these and not using the ones defined in the main python file?